dnf-json

```diff
@@ -198,46 +198,49 @@ def solve(request, cache_dir):
     arch = request["arch"]
```

Comment:
About the commit message: what (if anything) is the difference between this and depsolving the requests one after the other.

Reply:
We need this change to make chain-depsolving possible (that @thozza is working on). We need to be able to send multiple depsolve requests to dnf-json at the same time so that it can reuse the DNF Base object for subsequent depsolves.

Reply:
Right, that's what I thought. We should document this in the commit message or code, I guess. The way it reads now it appears they will be independent of each other.